Add first version of the application form

azettl opened the pull request.

azettl committed:
Add first version of the application form

JamesPaden commented:
Here's a new version without placeholders. Can you use this one instead?
[ExpectedBehavior_Templates_JobApp (1).pdf](https://github.com/DocRaptor/html-to-pdf-templates/files/11422507/ExpectedBehavior_Templates_JobApp.1.pdf)

JamesPaden committed:
Update job application templates

- Separate USLetter & A4 Styles
- Add checkbox and radio box examples
- Add public example requestbin URL for form submission testing
- Update documentation URLs

JamesPaden committed:
Add form screenshot to readme
